### ISSUE TITLE
Proxies need to merge with environment var

### ISSUE THREAD

**Issue by albertyckwok** [ravello/python-sdk#61]
Hi, I think we need to change the following line 244 of ravello_sdk.py to horner proxy setting in the environment variable as the SDK using load level call that doesn't check environment variables.
        self._proxies = {}
To:
        self._proxies = urllib.getproxies().get('http')

**Comment by hadard** [comment]
Hi @albertyckwok, this sounds like a nice enhancement. 
Feel free to send a pull request with this change 😄

**Comment by albertyckwok** [comment]
Hi @hadard ,

I have modified and tested the update. I have created the pull request. Let me know if you need anything else.

Thanks
Albert

**Comment by albertyckwok** [comment]
Sorry about that. I only tested in Python 2.7.5. I am guessing that you are using Python 3. According to https://docs.python.org/2/library/urllib.html the urllib module has been split into parts and renamed in Python 3.

**Comment by rut31337** [comment]
Yep, def a python 3 issue.  Even when I comment out this line, the SDK doesn't seem to work with Python3 any way.

**Comment by rut31337** [comment]
PR submitted https://github.com/ravello/python-sdk/pull/63
